aicsimageio/readers/ome_zarr_reader.py

```diff
@@ -135,6 +157,26 @@ def _get_coords(
 
         return coords
 
+    @staticmethod
+    def _get_pixel_size(
+        reader: ZarrReader, dims: List[str], series_index: int, resolution_index: int
+    ) -> Tuple[Optional[float], Optional[float], Optional[float]]:
+
+        coord_transform = reader.root_attrs["multiscales"][series_index]["datasets"][
+            resolution_index
+        ]["coordinateTransformations"]
+
+        spatial_coeffs = {}
+
+        for dim in ["X", "Y", "Z"]:
+            if dim in dims:
+                dim_index = dims.index(dim)
+                spatial_coeffs[str(dim)] = coord_transform[0]["scale"][dim_index]
```

Comment:
I think `dim` is always a `str` based on the list you are iterating

Reply:
resolved in [61ffab1](https://github.com/AllenCellModeling/aicsimageio/pull/509/commits/61ffab177495a28ebf821953b8604a7e5d6effed)